### ISSUE TITLE
[PromotionsBundle] Some dependencies are not found

### ISSUE THREAD

**Issue by marcaube** [Sylius/Sylius#1191]
Hi, I'm using – _well, trying to use_ – the PromotionsBundle as a standalone bundle to handle promotional codes in our checkout process ... and I've had some problems ...
### Configuration

I followed the [installation documentation](http://docs.sylius.org/en/latest/bundles/SyliusPromotionsBundle/installation.html) and was greeted with my first error when [updating the database schema](http://docs.sylius.org/en/latest/bundles/SyliusPromotionsBundle/installation.html#updating-database-schema) :

``` bash
[InvalidArgumentException]                                       
The class sylius.model.promotion_subject.class does not exists.
```

I resolved the issue by implementing the `PromotionSubjectInterface` and using my Entity as the `promotion_subject`. **This step should be documented.**

``` yml
sylius_promotions:
    driver: doctrine/orm
    classes:
        promotion_subject:
            model: Acme\OrderBundle\Entity\Order
```
### Backend integration

Next, I replaced the routes in `SyliusPromotionsBundle/Resources/config/routing.yml` with my own routes to override the templates and integrate the promotions in our backend.

``` yml
# src/Acme/BackendBundle/Resources/config/routing.yml

acme_backend_promotion_index:
    pattern: /promotions
    methods: [GET]
    defaults:
        _controller: sylius.controller.promotion:indexAction
        _sylius:
            template: AcmeBackendBundle:Promotion:index.html.twig
            sortable: true
            sorting:
                priority: desc

acme_backend_promotion_create:
    pattern: /promotions/new
    methods: [GET, POST]
    defaults:
        _controller: sylius.controller.promotion:createAction
        _sylius:
            template: AcmeBackendBundle:Promotion:create.html.twig
            redirect: sylius_backend_promotion_show

# ...
```

The `index` page went fine but the `create` page raised another error :

```
Could not load type "sylius_money"
500 Internal Server Error - InvalidArgumentException
```

So I searched for `sylius_money` and found it in [MoneyBundle](https://github.com/Sylius/Sylius/tree/master/src/Sylius/Bundle/MoneyBundle), which is not listed as a dependency in `composer.json`.

`sylius_money` is referenced in [FixedDiscountConfigurationType.php](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/PromotionsBundle/Form/Type/Action/FixedDiscountConfigurationType.php#L40) and in [ItemTotalConfigurationType.php](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/PromotionsBundle/Form/Type/Rule/ItemTotalConfigurationType.php#L40).
### TL;DR

I guess my question is : is there really a way to use this bundle as a standalone ?

**Comment by Rvanlaak** [comment]
The same `sylius_money` bug occurs here. We're also trying to only implement the promotions bundle. The workaround for now, until `SyliusMoneyBundle` is decoupled, is to add the bundle as a dependency?

**Comment by VaN-dev** [comment]
The same bug occured to me. So I tried to add the SyliusMoneyBundle to my composer to fix the bug. I created the sylius_money config, and i'm now facing another bug : Service of type "fixed_discount" does not exist
Because this service is not referenced in the PromotionBundle, but in CoreBundle. So PromotionBundle is dependant on CoreBundle ? If I install CoreBundle, it install the whole Sylius package. These bundles are definitely not dependant.. There is something wrong with it.
